src/sas/sasview/__init__.py

```diff
@@ -1,2 +1,6 @@
-__version__ = "4.2.0"
+__major_version__ = "4"
+__minor_version__ = "2"
+__version__ = __major_version__ + "." + __minor_version__ + ".1"
```

Comment:
If we are going to separate out versions, I think we should also do the point revision.
`__point_version__ = "1"`
`__version__ = __major_version__ + "." + __minor_version__ + "." + __point_version__ `

Reply:
indeed.  My real question is whether we need major and minor.  The reason is that the sphinx config uses 4.2 as the version number AND 4.2.1 as the release number -- which makes no sense to me and I have not found where that "version" number (4.2) is ever used.... but I didn't have the guts to remove it.   My feeling is we remove that? and get rid of major and minor here? or do as you say for full flexibility. Maybe something that can be discussed on a call?